src/components/ToolTip/__tests__/ToolTip.spec.js

```diff
@@ -0,0 +1,95 @@
+import { beforeEach, describe, expect, it, vi } from 'vitest';
+import { mount } from '@vue/test-utils';
+
+import ToolTip from '../ToolTip.vue';
+
+const createWrapper = (props) => {
+  return mount(ToolTip, { props });
+};
+
+describe('ToolTip', () => {
+  let wrapper;
+  beforeEach(() => {
+    wrapper = createWrapper({ text: 'Text', enabled: true });
+  });
+
+  it('should not display the tooltip when enabled is false', async () => {
+    await wrapper.setProps({ enabled: false });
+    const tooltipLabel = wrapper.find('[data-testid="tooltip-label"]');
+    expect(tooltipLabel.isVisible()).toBe(false);
+  });
+
+  it('should display the tooltip when enabled is true', async () => {
+    const tooltipLabel = wrapper.find('[data-testid="tooltip-label"]');
+    expect(tooltipLabel.isVisible()).toBe(true);
+  });
+
+  it('should force open the tooltip when forceOpen is true', async () => {
+    await wrapper.setProps({ forceOpen: true });
+
+    const tooltipLabel = wrapper.find('[data-testid="tooltip-label"]');
+    expect(tooltipLabel.isVisible()).toBe(true);
+  });
```

Comment:
You could also test here if `enable` is false, to ensure that the one making the tooltip visible is `forceOpen`